diff --git a/lib/providers.js b/lib/providers.js
index 6d6c4ea1b80499eecc9fcd138f8152466920ef09..12d75849d71085d0ba02803cf1631b8bde3ef839 100644
--- a/lib/providers.js
+++ b/lib/providers.js
@@ -26,7 +26,7 @@ export default function parseProviders(params) {
 // We should return both a client and authorization server config.
 function normalizeOAuth(c) {
     if (c.issuer)
-        c.wellKnown ?? (c.wellKnown = `${c.issuer}/.well-known/openid-configuration`);
+        c.wellKnown || (c.wellKnown = `${c.issuer}/.well-known/openid-configuration`);
     const authorization = normalizeEndpoint(c.authorization, c.issuer);
     if (authorization && !authorization.url?.searchParams.has("scope")) {
         authorization.url.searchParams.set("scope", "openid profile email");